src/SDKs/DBforPostgreSQL/Management.DBforPostgreSQL/Microsoft.Azure.Management.DBforPostgreSQL.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.DBforPostgreSQL</PackageId>
+		<Description>Provides developers with libraries for the Azure Database for PostgreSQL under Azure Resource manager to deploy an instance of PostgreSQL. Create, Update, Delete, Restore, and more.</Description>
+		<Version>0.1.0</Version>
```

Comment:
@rohit-joy as this is your first version, we recommend you release it as preview package and use 0.9.0-preview as your nuget version and 0.9.0 as your assembly/assemblyfile version